### PR TITLE
Feature/responsive sass structure

### PR DESCRIPTION
**Pull request by Andrey-1992**
Css styling and SASS

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -358,88 +361,241 @@ p {
   text-align: center;
   font-size: 125%;
   margin-bottom: 1%;
-  // font-family: $cursive-handle-font-family;
 }
 
 .planning-cost {
   text-align: center;
-  font-size: 135%;
+  font-size: 105%;
```
Comment (e-spitz) [inline src/css/base.scss:368]: These are huge percentages! You could try using 'ems' instead of '%' !

File: src/css/base.scss [hunk 2]
```diff
@@ -358,88 +361,241 @@ p {
...
-  // top: 50%;
-  // left: 50%;
-  // transform: translate(-50% , -50%);
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  grid-gap: 2%;
+  margin-top: 5%;
+  margin-left: 15%;
 }
```
Comment (e-spitz) [inline src/css/base.scss:412]: Nice use of flex on all of these stylings rather than position absolute

**Review by e-spitz — COMMENTED** [review]
Nice work Andrey! Can't wait to see the finished product.

File: src/css/base.scss (hunk 1 above)
Comment (Andrey-1992) [reply src/css/base.scss:368]: Thank you, I will work on it !

**Pull request merged**